.github/workflows/ci.yml

```diff
@@ -0,0 +1,206 @@
+name: ACS Community Packaging CI
+
+on:
+  pull_request:
+    branches:
+      - master
+      - release/**
+      - feature/**
+      - fix/**
+  push:
+    branches:
+      - master
+      - release/**
+      - feature/**
+      - fix/**
+  workflow_dispatch:
+
+env:
+  GIT_USERNAME: ${{ secrets.BOT_GITHUB_USERNAME }}
+  GIT_EMAIL: ${{ secrets.BOT_GITHUB_EMAIL }}
+  GIT_PASSWORD: ${{ secrets.BOT_GITHUB_TOKEN }}
+  DOCKERHUB_PASSWORD: ${{ secrets.DOCKER_PASSWORD }}
+  DOCKERHUB_USERNAME: ${{ secrets.DOCKER_USERNAME }}
+  GITHUB_ACTIONS_DEPLOY_TIMEOUT: 60
+  MAVEN_PASSWORD: ${{ secrets.NEXUS_PASSWORD }}
+  MAVEN_USERNAME: ${{ secrets.NEXUS_USERNAME }}
+  QUAY_PASSWORD: ${{ secrets.QUAY_PASSWORD }}
+  QUAY_USERNAME: ${{ secrets.QUAY_USERNAME }}
+  TAS_ENVIRONMENT: ./tests/environment
+  TAS_SCRIPTS: ../alfresco-community-repo/packaging/tests/scripts
+  # Release version has to start with real version (7.3.0-....) for the docker image to build successfully.
+  RELEASE_VERSION: 7.2.1
```

Comment:
Shouldn't it be 7.2.2?

Reply:
Or even 7.2.2-A1?

Reply:
I was planning on changing it to 7.2.2-A1 in the release PR. This PR is just for the conversion to GHA